pallets/pips/src/lib.rs

```diff
@@ -703,7 +731,9 @@ decl_module! {
         /// * `proposal` a dispatchable call
         /// * `deposit` minimum deposit value, which is ignored if `proposer` is a committee.
         /// * `url` a link to a website for proposal discussion
-        #[weight = (1_850_000_000, DispatchClass::Operational, Pays::Yes)]
+        // TODO: Use `propose_from_committee` if the origin is that of the committee.
+        // TODO: Get the proposal length and use that instead of the constant.
+        #[weight = <T as Trait>::WeightInfo::propose_from_community(1_000)]
```

Comment:
Instead of hardcoding `1_000`, please use the actual value. Something like the length of the `proposal` should be good enough.